### PR TITLE
Added Quiz

### PR DESCRIPTION
**Pull request by SyntappZ**
Added a quiz section, we just need some more questions added, they can be added in the quizQuestions.js file 🙂👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ Here we'll add some examples...
 - add an html element, for example: ``` <h1>Open Source Rules</h1> ```
 - add some js, for example: ```logHello()```
 
+## Add to Quiz
```
Comment (DavidMatthewFraser) [inline README.md:32]: Thanks for updating the README with the new changes

File: assets/js/quizQuestions.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+const questionTemplate = {
+  question: "",
+  answers: ["", "", "", ""],
+  correctAnswer: "",
+};
+
+// feel free to add some questions to the arrays a template is above, you can put more of less than for answers but dont have to many
+
+
+const htmlQuestions = [
```
Comment (DavidMatthewFraser) [inline assets/js/quizQuestions.js:10]: I like how you made it really easy to add questions

File: index.html [hunk 3]
```diff
@@ -59,10 +59,13 @@ <h3>Easy to HTML</h3>
         </div>
 
         <div class='row'>
-          <div class='col-md-6 col-lg-6 col-sm-6 col-xs-6'>
+          <div class='col-md-6 col-lg-6 col-sm-12 col-xs-12'>
             <a href='puzzles/puzzle1.html'><button class='btn btn-default btn-block'>Puzzle One Prototype</button></a>
           </div>
-          <div class='col-md-6 col-lg-6 col-sm-6 col-xs-6'>
+          <div class='col-md-6 col-lg-6 col-sm-12 col-xs-12'>
```
Comment (DavidMatthewFraser) [inline index.html:65]: Thanks for updating index.html with the changes

File: quiz/quiz.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```
Comment (DavidMatthewFraser) [inline quiz/quiz.html:1]: Such clean design! Very impressive 👌👌

**Comment by DavidMatthewFraser** [comment]
The quiz is sick! Thanks for the time/hardwork/creativity. 
You've put a lot into this project, let me know if you're interested in being an administrator for the repo.

File: assets/js/quizQuestions.js (hunk 2 above)
Comment (SyntappZ) [reply assets/js/quizQuestions.js:10]: Yeah i try to make it as simple as possible, thats also why i put in separate file so people can find easy, will be good when some more questions are added and its a proper quiz.

**Comment by SyntappZ** [comment]
thanks man appreciate that, yh sure il be an admin but i probably wont need to be, but still sound cool 😉👌

**Pull request merged**